docs/data/README.mdx

```diff
@@ -2,10 +2,27 @@
 title: Data Overview
 ---
 
-The RPC, data indexers, Hubble, and Horizon are platforms or tools that facilitate communication between users and applications and the Stellar network. With these various platforms, users can query network data, submit transactions, and interact with smart contracts without needing to understand the low-level details of Stellar Core.
+Stellar supports tools that facilitate communication between users and applications on the Stellar network. Users can query network data, submit transactions, and interact with smart contracts without needing to understand the low-level details of Stellar Core.
 
 This section will walk you through the differences between the various platforms and tools, what platform or tool is best for what use case, and then link to their various documentation locations.
 
+- **RPC** - live network gateway
+- **Horizon** - API for network state data
+- **Galexie** - exports raw ledger metadata files
+- **Hubble** - OLAP database for network data
```

Comment:
dumb question: Do you think enough of the readership will understand OLAP? If not, we may want to be more explicit and say either 'modern data warehouse' or 'analytics database'

Reply:
Good point. I like "analytics database"

Reply:
updated in e250dd6664b5c0a62fb1d27846ac2633cd0ae9f7